CONTRIBUTING.md

```diff
@@ -298,18 +302,43 @@ When you edit a blank section:
 
 - If your PR adds a **breaking change**, create a new heading called `#### BREAKING CHANGES` and add
   bullet point that explains the breaking changes; provide a migration path if possible.
-- If your PR adds a new feature of a fix, create a new heading called `#### Other changes` and
-  add a bullet point that explains the fix or the new feature. Make sure that this new heading
-  appears after the `#### BREAKING CHANGES` heading.
+  Read [how we version Biome](https://biomejs.dev/internals/versioning/) to determine if your change is a breaking change.
+  A breaking change result in a major release.
+- If your PR adds a new feature, enhance an existing feature, or fix a bug, create a new heading called `#### New features`, `#### Enhancements`, or `#### Bug fixes`.
+  Then, add a bullet point that explains the change.
+
+Make sure that the created subsections are ordered in the following order:
+
+```md
+#### BREAKING CHANGES
+
+#### New features
+
+#### Enhancements
+
+#### Bug fixes
+```
+
+Bullet points should be sorted in their importance order: important changes come first.
+
+Because the website displays the changelog,
+you should update the website using the following command:
+
+```sh
+cargo codegen-website
```

Comment:
```suggestion
just codegen-website
```

Let's use `just`; that's the tool we're telling contributors to use. If we don't have a script for that, let's create one